### ISSUE TITLE
[1.5][react-cat] Market picker tooltip should not hide list

### ISSUE THREAD

**Issue by froooze** [bitshares/bitshares-ui#2931]
**Describe the bug**
The tooltip hides the options list

**To Reproduce**
1. Go Market picker
2. Search
3. Hover over names
4. See tooltip

**Expected behavior**
Show tooltip next to the market picker or completely remove tooltip

**Screenshots**
![Market-picker-tooltip](https://user-images.githubusercontent.com/40874087/61091989-a4230280-a444-11e9-9071-5809ab73c60b.png)

**Comment by react-cat** [comment]
@froooze If we will show tooltip below or on the side of market picker it might be bad UX cause on small screens it will get of the screen or simply user wouldn't see it in case it will appear below picker. 

As a suggestion we can create a tooltip that will appear on the right side next to asset name and will contain first N symbols of current tooltip and three dots. There are few variants of what would happen when user click on dots: 
1. user will be redirected to the asset page 
2. tooltip will expand and user will see full info. 

Also as another suggestion, we can simply remove tooltip and just redirect user to asset page after click on asset title

**Comment by startailcoon** [comment]
I'd say we should rather just remove the tooltip on that place. However, since the reword I still think we're missing a link to the asset details page. 

The modal should rather have a Table structured something like the following in my opinion. It would also make it possible to sort it.

Asset|Owner|Info|Market
-|-|-|-
gdex.EOS|gdex-team|[Link]|[Link]
gdex.EOS|gdex-team|[Link]|[Link]
gdex.EOS|gdex-team|[Link]|[Link]

**Comment by froooze** [comment]
@startailcoon:
- the info link column can be removed, when the asset name has the info link
- now the Asset column has the link to the market
- Market column should be named Trade, like the other columns

**Comment by startailcoon** [comment]
> 
> 
> @startailcoon:
> Do we really need the market column? By clicking on Asset or Owner column the asset is selected and the markets opens afterwards automatically.

Sorry, of course you're right. We do not need the market column. But since we link to the market by clicking on the asset or owner today, I think that a asset info link is in order. And it would need some prettying up, I think we can do better.

**Comment by react-cat** [comment]
@startailcoon so is there anything to do here since you created a related task that will solve this issue ?

**Comment by startailcoon** [comment]
@react-cat this issue would rather be to style the results list as a table. Since the popup is much larger than just the market picker I created a separate issue for it.

**Comment by startailcoon** [comment]
Was thinking on this, and we could add the asset details popup on a "?" symbol at the beginning of the row. It would be good for the user to be able to read the details quick, but it shouldn't be when hovering the part where you actually select the market.

**Comment by startailcoon** [comment]
@react-cat interested in taking this task?

**Comment by react-cat** [comment]
@startailcoon Yes, but  actually I can't find a button to open market picker and please, can you summarize a task for me not to lose any details ?

**Comment by startailcoon** [comment]
@react-cat of course. Summarizing all this made me realize that it may need some more time. I've increased it to be 1.5 hrs. Mainly because table pagination and the starred asset handling requires some work. Also note that the `<Asset ..>` link we use today still does not have an option not to view the popup. But I'm working on that on the side on the other issue.

Market picker can be found by clicking on the asset names of the market pair on the Exchange. (note `1` below)

In the market picker window I would like to change these things. (Note `2` below).

**New table structure**

⭐|Asset|Owner|Info
-|-|-|-
⭐|gdex.EOS❓|gdex-team|[Link]
⭐|open.EOS❓|gdex-team|[Link]
-|gdex.BTC❓|gdex-team|[Link]

1. The list should be a table, structured like mentioned above. It should have pagination if required and display 10 assets per page to not make it to long.
2. No scrollbar
3. When nothing is searched for it should use the defaultValue that all other tables use when there is nothing to view, just look on other tables. 

First column is starred if the user has the asset as a favorite. For now we should use the list from the landing page, which is added from the search market (note `3` below). It could be refined later. Starred are wildcards, so for example `EOS` will be `*EOS*` so it will match any EOS asset. This column has no action.

Second column is the asset name, behind it is a `?` which will display the asset information details. This columns action will be to change asset pair.

Third column is asset creator. Column action is same as second column.

Fourth column will link to the asset information page. I don't know what an appropriate icon would be here. Perhaps a button would do as well. Could be `[Asset Info]`.

# 1 
![image](https://user-images.githubusercontent.com/12114550/62160875-47b15580-b315-11e9-8c37-939072cae551.png)

# 2 
![image](https://user-images.githubusercontent.com/12114550/62161038-a1b21b00-b315-11e9-809d-9935fd2325b5.png)

# 3

### Landing Page
![image](https://user-images.githubusercontent.com/12114550/62161789-3bc69300-b317-11e9-8bff-8a7cec8c0152.png)

### On Exchange Page
![image](https://user-images.githubusercontent.com/12114550/62161996-965fef00-b317-11e9-9898-8ec09313efe6.png)

**Comment by sschiessl-bcp** [comment]
What do you want to say with the pictures?

**Comment by startailcoon** [comment]
> 
> 
> What do you want to say with the pictures?

See my notes in the text with numbers.

**Comment by startailcoon** [comment]
@react-cat Please note my update. I had a second thought on the starred flag. It should be based on starred markets. 

Say that you have BTS/GDEX.EOS as a starred market, and you want to switch from BTS/OPEN.BTC.
If you want to replace OPEN.BTC, and search for EOS, then GDEX.EOS would be starred as it matches your favorite BTS/GDEX.EOS.

**Comment by react-cat** [comment]
@startailcoon Okay, got it

**Comment by react-cat** [comment]
@startailcoon should user have an ability to select market as favorite on market picker ?

**Comment by react-cat** [comment]
@startailcoon 
"Second column is the asset name, behind it is a ? which will display the asset information details. This columns action will be to change asset pair."
By asset information details you mean same tooltip that was here before ? 

And also I think we can use Ant info-circle icon, cause a lot of icons might cause an UI overload. They will attract too much attention

**Comment by sschiessl-bcp** [comment]
This task is to ensure that the tooltip doesn't break the functionality. Please stick to that, and use new issue for restyling.

**Comment by react-cat** [comment]
@startailcoon Due to @sschiessl-bcp comment, let's decide what we are going to leave for that task and create a new one. Also maybe we could close this task and open a task for redesign, cause it would solve current problem and there is no need to finish this task just to make a fix for few days that will be deleted in redesign task. Furthermore, I am already working on that task, so if we decide to split this task and firstly finish this one, it might cause merge conflicts when I come back to redesign.

**Comment by startailcoon** [comment]
@react-cat @sschiessl-bcp sorry, I'll take the blame here. The tooltip issue was actually moved to #2978 since it's not an isolated issue part of the market picker. 

Therefor I used this issue as a means to do some styling issues. With the end result being confusing I agree with us making a new issue for just the restyling of the market picker and closing this issue without any work needed to complete.
